src/NumericalAlgorithms/LinearOperators/Divergence.hpp

```diff
@@ -39,7 +39,7 @@ namespace Tags {
 /// Prefix indicating the divergence of a Tensor or that a Variables
 /// contains divergences of Tensors.
 ///
-/// \see Tags::DivVariablesCompute
+/// \see Tags::DivVariablesCompute Tags::DivVectorCompute
```

Comment:
Swap order of these two, to make the order of the examples match the order in previous sentence